gradle-8.yaml

```diff
@@ -1,15 +1,13 @@
 package:
   name: gradle-8
-  version: 8.0.2 # When bumping this, also bump the provides below!
-  epoch: 2
+  version: 8.1.1 # When bumping this, also bump the provides below!
```

Comment:
I know the comment wasn't added in this PR, but I'm not sure what to make of it. Does the update automation handle updating the provides? If so I'd say we can remove the comment. If the automation doesn't handle this, and it's enabled right now, this seems like a footgun. WDYT?

Reply:
I added the comment, in an attempt to at least warn about the footgun.

Ideally we could use vars in provides and it'd stay up to date itself. I haven't made that change in melange yet though.

Reply:
The updater doesn't as AFAIK. @kaniini @imjasonh to check that I'm doing it right.

Reply:
I think the comment is good @imjasonh 👍  — perhaps "footgun" is too strong, I just meant that I could see someone (like myself) overlooking this and merging a messed up state.